source/extension/F0.Analyzers.Vsix/F0.Analyzers.Vsix.csproj

```diff
@@ -25,7 +25,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.0.5234" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.0.5240" PrivateAssets="all" />
```

Comment:
I'm quite uncertain about the _Visual Studio SDK_:
I choose the latest `v17.0.*`, because `17.0` is the minimal version of the Roslyn compiler supported by the NuGet package, see `F0.Analyzers.csproj`:
```xml
<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.0.1" PrivateAssets="all" />
```
See also https://github.com/dotnet/roslyn/blob/main/docs/wiki/NuGet-packages.md

Reply:
latest version looks good enough. What information would resolve you being uncertain?

Reply:
If we build the VSIX with the latest `Microsoft.VSSDK.BuildTools` (currently `17.7.2196`), would the extension still be compatible with Visual Studio 2022 `v17.0`, or only with `v17.7`?
I'd like to keep the requirements of both the VSIX extension and the NuGet package compatibility on a par.
I don't have a `v17.0` installation around to try it out.